active/teachingtip/TeachingTipSpec.md

```diff
@@ -0,0 +1,565 @@
+# Description
+
+A teaching tip is a semi-persistent and content-rich flyout that provides contextual information. It is often used for informing, reminding, and teaching users about important and new features that may enhance their experience.
+
+**Important APIs:** [TeachingTip class]() [Link TODO]
+
+A teaching tip may be light-dismiss or require explicit action to close. A teaching tip can point at an element or be used without an element to point at.
+
+### Is this the right control? 
+
+Use a **TeachingTip** control to focus a user's attention on new or important updates and features, remind a user of nonessential options that would improve their experience, or teach a user how a task should be completed. 
+
+Because teaching tip is transient, it would not be the recommended control for prompting users about errors or important status changes.
+
+
+# Examples
+
+### Some APIs shown in the following examples are not yet available in the prerelease version of this control. Please see the [Teaching Tip Sample in the XAML Sample Gallery](https://github.com/Microsoft/Xaml-Controls-Gallery/blob/w/saschule/TeachingTipSample/XamlControlsGallery/ControlPages/TeachingTipPage.xaml) for implementation examples that are accurate with the version of this control currently available through prerelease. 
+
+
+The teaching tip has several states, including these notable ones.
+
+A teaching tip can point to a specific UI element to enhance contextual clarity of the information it is presenting. 
+
+![A sample app with a teaching tip pointing at the save button. The tip title reads "Saving automatically" and the subtitle reads "We save your changes as you go - so you never have to." There is a close button on the top right corner of the teaching tip.](TeachingTipPointerSample.jpg)
+
+A teaching tip can be used without a pointer when the information that it is presenting is not specific to a UI element. 
```

Comment:
Wikipedia says that the pointy thing in a speech balloon is called a "[tail](https://en.wikipedia.org/wiki/Speech_balloon#Speech_bubbles)". This agrees with the terminology used in [our own UI guidelines text](https://docs.microsoft.com/en-us/windows/desktop/uxguide/ctrl-balloons).

Reply:
Fixed by both this [commit ](https://github.com/Microsoft/microsoft-ui-xaml-specs/commit/e28ae9aed3521b2ba28f107fb6c11e73cced415f) and [this ](https://github.com/Microsoft/microsoft-ui-xaml-specs/commit/76b323b15c5d313e086c4702e6275c510360df8b) one.